unstructured/staging/base.py

```diff
@@ -337,3 +338,96 @@ def filter_element_types(
         return filtered_elements
 
     return elements
+
+
+def convert_to_coco(
+    elements: List[Element],
+    dataset_description: Optional[str] = None,
+    dataset_version: str = "1.0",
+    contributors: Tuple[str] = ("Unstructured Developers"),
```

Comment:
Add a comma in the parentheses to make it a tuple, otherwise Python thinks these are just order-of-operations parentheses, not tuple-parentheses.
```suggestion
    contributors: Tuple[str] = ("Unstructured Developers",),
```